### PR TITLE
KAA-635: Minor nitpicks addressed

### PR DESCRIPTION
**Pull request by RostakaGmfun**
/cc @rasendubi

### CODE REVIEW

File: client/client-multi/client-c/src/kaa/platform/ext_key_utils.h [hunk 1]
```diff
@@ -82,7 +82,7 @@ kaa_error_t ext_encrypt_data(const uint8_t *input, size_t input_size, uint8_t *o
  * @param[out] output_payload_size  The size of the payload data.
  * @param[out] output               The buffer which contains dectypted data will be filled in.
  *
- * @return error code.
+ * @retval KAA_ERR_NONE on success. The other value means failure.
```
Comment (akokhanovskyi) [inline client/client-multi/client-c/src/kaa/platform/ext_key_utils.h]: "The other" means some specific. "Any other" is more appropriate.

File: client/client-multi/client-c/src/kaa/platform/ext_key_utils.h [hunk 2]
```diff
@@ -95,7 +95,7 @@ kaa_error_t ext_decrypt_data(const uint8_t *input, size_t input_size,
  * @param[in]   remote_key          Remote public key which will be used to encrypt session key.
  * @param[in]   remote_key_size     Remote public key's size.
  *
- * @return error code.
+ * @retval KAA_ERR_NONE on success. The other value means failure.
```
Comment (akokhanovskyi) [inline client/client-multi/client-c/src/kaa/platform/ext_key_utils.h]: Same as above

File: client/client-multi/client-c/src/kaa/platform/ext_key_utils.h [hunk 3]
```diff
@@ -116,6 +116,7 @@ size_t ext_get_encrypted_data_size(size_t input_size);
  * @param [out] output           The pointer which will be initialized with signed key.
  * @param [out] output_size      The length of signed key.
  *
+ * @retval KAA_ERR_NONE on success. The other value means failure.
```
Comment (akokhanovskyi) [inline client/client-multi/client-c/src/kaa/platform/ext_key_utils.h]: Same as above

File: client/client-multi/client-c/src/kaa/platform/ext_key_utils.h (hunk 1 above)
Comment (rasendubi) [inline client/client-multi/client-c/src/kaa/platform/ext_key_utils.h]: It should go like:

```
@return Error code.
@retval KAA_ERR_NONE Success.
```

Actually, it's much better to document some return values, as the current documentation is true for all functions returning `kaa_error_t` -- that doesn't add much value but rather worsens things.

**Comment by rasendubi** [comment]
My comment is still not addressed.

**Comment by rasendubi** [comment]
LGTM

**Comment by forGGe** [comment]
+

**Pull request merged**